packages/flex-plugin-e2e-tests/src/tests/step005.ts

```diff
@@ -14,28 +14,28 @@ const testSuite: TestSuite = async ({ scenario }: TestParams): Promise<void> =>
   const ext = scenario.isTS ? 'tsx' : 'jsx';
   await replaceInFile({
     files: joinPath(plugin.dir, 'src', 'components', 'CustomTaskList', `CustomTaskList.${ext}`),
-    from: /This is a dismissible demo component.*/,
+    from: /This is a dismissible demo component./,
     to: plugin.componentText,
   });
 
-  const envVariableValue = `${Date.now()}`;
+  // const envVariableValue = `${Date.now()}`;
 
-  writeFileSync(`${plugin.dir}/.env`, `FLEX_APP_ENVIRONMENT_TEST=${envVariableValue}`);
+  // writeFileSync(`${plugin.dir}/.env`, `FLEX_APP_ENVIRONMENT_TEST=${envVariableValue}`);
 
-  await replaceInFile({
-    files: joinPath(plugin.dir, 'src', 'components', 'CustomTaskList', `CustomTaskList.${ext}`),
-    from: /close.*/,
-    to: `close-{process.env.FLEX_APP_ENVIRONMENT_TEST}`,
-    countMatches: true,
-  });
+  // await replaceInFile({
+  //   files: joinPath(plugin.dir, 'src', 'components', 'CustomTaskList', `CustomTaskList.${ext}`),
+  //   from: /close.*/,
+  //   to: `close-{process.env.FLEX_APP_ENVIRONMENT_TEST}`,
+  //   countMatches: true,
+  // });
 
   await spawn('twilio', ['flex:plugins:build', '-l', 'debug'], { cwd: plugin.dir });
 
   assertion.not.dirIsEmpty([plugin.dir, 'build']);
   assertion.fileExists([plugin.dir, 'build', `${plugin.name}.js`]);
   assertion.fileExists([plugin.dir, 'build', `${plugin.name}.js.map`]);
   assertion.fileContains([plugin.dir, 'build', `${plugin.name}.js`], plugin.componentText);
-  assertion.fileContains([plugin.dir, 'build', `${plugin.name}.js`], envVariableValue);
+  // assertion.fileContains([plugin.dir, 'build', `${plugin.name}.js`], envVariableValue);
```

Comment:
This is still a valid test case?

Reply:
No, we don't have close icon in the 2.x template, hence removed this.